packages/coding-standards/composer.json

```diff
@@ -11,22 +11,23 @@
         }
     ],
     "conflict": {
-        "friendsofphp/php-cs-fixer": "2.19.0"
+        "friendsofphp/php-cs-fixer": ">=3.3.2"
     },
     "require": {
         "php": "^7.4 || ^8.0 || ^8.1",
         "ext-tokenizer": "*",
-        "friendsofphp/php-cs-fixer": "^2.17.0",
         "nette/utils": "^3.1.3",
-        "object-calisthenics/phpcs-calisthenics-rules": "^3.1",
-        "php-parallel-lint/php-parallel-lint": "^1.3.1",
-        "slevomat/coding-standard": "^6.3.5",
-        "squizlabs/php_codesniffer": "^3.5.0",
-        "symplify/easy-coding-standard": "^8.3.48",
         "symfony/finder": "^4.4|^5.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9.5.20"
+        "friendsofphp/php-cs-fixer": "^3.3.1",
```

Comment:
nemam paru, jak funguje symplify monorepu builder, ale treba na uxf/fan jsem se na tyhle dev deps uplne vykaslal a jsou jen v root composer.json

Reply:
In this case packages should be defined because we have custom rules.